Realm/RLMSyncManager.h

```diff
@@ -79,14 +79,6 @@ typedef void(^RLMSyncErrorReportingBlock)(NSError *, RLMSyncSession * _Nullable)
 @property (nonatomic, copy) NSString *appID;
 
 /**
- Whether SSL certificate validation should be disabled. SSL certificate validation is ON by default. Setting this
```

Comment:
They're not hard to make at all, but it would really be good to move users off this global property (and minimize the risk that they disable validation for their entire application unintentionally).